### DISCUSSION TITLE
Save last-used multiplayer lobby filters when opening the multi screen

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#26218] [Ideas] [answered]
Right now, whenever a player opens the multi menu to view lobbies, the filter defaults to "all" and "open". As this happens any time I back out of the menu then back in (even if I change the filter settings), I have change the filter to my preferences (e.g., "public" instead of "all", since I don't like the private lobbies cluttering the list). It would be nice if the last used filters were saved upon backing out of the menu, so that it has them set next time I open the multi menu (i.e., if I exclude private lobbies, back out of the menu, then go back in, the list still excludes private lobbies until I decide to change the filter myself).

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26538
